packages/babel-preset-env/test/fixtures/corejs2/usage-shippedProposals/output.js

```diff
@@ -1,33 +1,39 @@
 "use strict";
 
-require("core-js/modules/es6.object.define-properties.js");
+require("core-js/modules/es6.object.to-string.js");
 
-require("core-js/modules/es7.object.get-own-property-descriptors.js");
+require("core-js/modules/es6.promise.js");
 
-require("core-js/modules/es6.array.for-each.js");
+require("core-js/modules/es6.symbol.js");
 
-require("core-js/modules/es6.array.filter.js");
+require("core-js/modules/es7.symbol.async-iterator.js");
 
-require("core-js/modules/es6.object.define-property.js");
+require("core-js/modules/es6.object.keys.js");
 
 require("core-js/modules/es6.array.index-of.js");
 
-require("core-js/modules/web.dom.iterable.js");
+require("core-js/modules/es6.object.define-property.js");
 
-require("core-js/modules/es6.array.iterator.js");
+require("core-js/modules/es6.array.filter.js");
 
-require("core-js/modules/es6.object.keys.js");
+require("core-js/modules/es6.object.get-own-property-descriptor.js");
 
-require("regenerator-runtime/runtime.js");
+require("core-js/modules/es6.array.for-each.js");
 
-require("core-js/modules/es7.symbol.async-iterator.js");
+require("core-js/modules/es7.object.get-own-property-descriptors.js");
 
-require("core-js/modules/es6.symbol.js");
+require("core-js/modules/es6.object.define-properties.js");
 
-require("core-js/modules/es6.promise.js");
+require("regenerator-runtime/runtime.js");
 
 require("core-js/modules/es6.object.to-string.js");
 
+require("core-js/modules/es6.promise.js");
+
+require("core-js/modules/es6.symbol.js");
+
+require("core-js/modules/es7.symbol.async-iterator.js");
```

Comment:
Duplicate polyfill imports should have been avoided.

Reply:
Fixed by https://github.com/babel/babel-polyfills/pull/55